environment/bashrc/.bashrc_cray

```diff
@@ -90,7 +90,7 @@ fi
 if [[ ${CRAY_CPU_TARGET} =~ haswell ]] || [[ ${CRAY_CPU_TARGET} =~ knl ]]; then
 
   module use --append ${VENDOR_DIR}-ec/modulefiles
-  dracomodules="git cmake numdiff gsl/2.5 metis eospac/6.4.0 random123 parmetis \
+  dracomodules="git cmake/3.17.0 numdiff gsl/2.5 metis eospac/6.4.0 random123 parmetis \
```

Comment:
Is 3.17 in friendly testing on the cray machines? If so, out of curiosity what is the default version (if you just module load cmake)?